Resources/Private/Templates/Publication/List.html

```diff
@@ -39,13 +43,13 @@
 <f:section name="GroupTitle">
 	<f:switch expression="{filter.groupby}">
 		<f:case value="0">
-			<publications:group.title title="{publication.year}" before="<h4>" after="</h4>" />
+			<publications:group.title title="{publication.year}" before='<h4 id="c{publication.year}">' after="</h4>" />
```

Comment:
An anchor with a c and a year is not unique. There could also be a content element with uid 2021 e.g. on the same page and that would have the same anchor.

Reply:
I agree,  though unlikely unless multiple plugins are placed on same page. 
Do you have a recommendation or a policy for this plugin how to make it unique? We could append it with the plugin name (i.e. c2010-publications), but this would not resolve the double instance of plugins on same page.